### PR TITLE
chore: try renaming package without org

### PR DESCRIPTION
**Pull request by orvn**
@tikagan can you look into unpublishing and republishing, but without the org name?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@undone-labs/fontpls",
```
Comment (orvn) [inline package.json:2]: `npm i -g undone-labs/fontpls` is a little lengthy (what's in the readme is ideal)

Comment (tikagan) [reply package.json:2]: @orvn I can unpublish and republish without the org name but that would publish it not under the undone-labs organization on npm. The format `@<org-name>/<package-name>` for the name in the `package.json` seems to be the thing that actually scopes the npm package under the org

Comment (tikagan) [reply package.json:2]: @orvn ping ^

Comment (tikagan) [reply package.json:2]: Note: in the ticket comments it was decided that the package would be unpublished from npm under the `undone-labs` organization and republished by @orvn

**Review by tikagan — APPROVED** [review]
lgtm!

**Pull request merged**